oracle/src/main.cpp

```diff
@@ -3,6 +3,7 @@
 #include <QIcon>
 #include <QTranslator>
 #include <QLibraryInfo>
+#include <QCommandLineParser>
```

Comment:
I'm leaving the changes in Oracle as an Easter egg that can be used in the future.

Reply:
>the first attempt of this PR used Oracle to download the spoilers. Since I find that code to be useful, I have decided to leave it in the PR as an Easter Egg so it can be used in the future by another process (who knows, maybe we'll want to have Oracle do new things, and this is a great example of what it is capable of!)

If I understand it correctly you added unused code, shouldn't you comment that accordingly in the code at least?
The reference in this pr will be forgotten and the next coder will wonder. :)

Reply:
There's nothing to comment really. It's fairly obvious this code isn't used (as there are no calls to it).